AsyncDisplayKit/ASVisualEffectNode.h

```diff
@@ -0,0 +1,41 @@
+//
+//  ASVisualEffectNode.h
```

Comment:
You'll want to include this header in AsyncDisplayKit.h so that it can be used outside of the framework. Also, update the copyright preamble to reflect the one used in the other classes.